modules/local/compute_test.nf

```diff
@@ -14,6 +14,7 @@ process COMPUTE_TEST {
 
     script:
     """
+    export TMPDIR="./"
```

Comment:
Nit: Without reading much more about it, can you let me know if the nextflow [scratch](https://www.nextflow.io/docs/latest/process.html#scratch) directive is relevant here?

Reply:
When I wrote this, I looked into `scratch`, but it's not relevant. It has to do with changing where Nextflow is creating the work directory, whereas my goal is to change where Python is creating its temporary directories.